odl/solvers/functional/functional.py

```diff
@@ -912,35 +924,58 @@ def __init__(self, left, right):
 
     @property
     def gradient(self):
-        """Gradient operator of the functional."""
+        """Gradient operator of the functional.
+
+        Notes
+        -----
+        The derivative is computed using Leibniz's rule:
+
+        .. math::
+
+            [D(f g)](p) = g(p) [Df](p) + f(p) [Dg](p)
```

Comment:
Ah, you mean I should use code style? Just thought that is looked better in math typesetting but could change that. Probably wont fit on a line though.